alacritty/src/main.rs

```diff
@@ -62,6 +61,10 @@ use crate::event::{Event, Processor};
 #[cfg(target_os = "macos")]
 use crate::macos::locale;
 
+// TODO refactor to use `ExitCode` when bumping minimum supported rust version to 1.61.
+pub const EXIT_FAILURE: i32 = 1;
+pub const EXIT_SUCCESS: i32 = 0;
```

Comment:
I don't think these constants provide any benefit at all in the current code.

Reply:
See the comment wrt `run_return`.